docs/content/en/integrations/social-authentication.md

```diff
@@ -315,20 +317,22 @@ Newly created users are neither staff nor superuser by default. The `is_staff` f
 
 `.*@example.com` will make `alice@example.com` a staff user, while `bob@partner.example.com` or `chris@example.org` will be non-staff users.
 
-
 ## Login speed-up
 
-If you are using only one Social authentication and you are not using the standard login mechanism (`SHOW_LOGIN_FORM` is
-set to `False`), showing login page could be useless because every time user clicks on the only existing button on the
-page like "Login with SAML" (or another similar button). If you set `SOCIAL_LOGIN_AUTO_REDIRECT` to `True`, the login
-page is skipped and the user is automatically redirected to the identity provider's page.
+You can bypass the login form if you are only using SSO/Social Auth for logging in by enabling the two environment variables:
+
+```
+DD_SOCIAL_LOGIN_AUTO_REDIRECT: "true"
+DD_SOCIAL_AUTH_SHOW_LOGIN_FORM: "false"
+```
+
+These values can also be adjust in `dojo/settings/settings.dist.py`
```

Comment:
Configurations shall not be changed in `settings.dist.py` directly. We have a documentation page how to do it. Please remove that line.